admin-manual/installation-setup/customization/dashboard-config.rst

```diff
@@ -710,22 +711,69 @@ user, you can change it via the command-line:
+
+Archivematica can mint persistent identifiers (PIDs) for digital objects,
+directories, or AIPs by defining the PIDs in a configured `Handle.Net`_
+registry. Handle.Net can then create persistent URLs (PURLs) from the PIDs and
+can reroute requests to the persistent URLs to a target URL that is configured
+in Handle.Net.
+
+.. image:: images/handlenet-config.*
+   :align: right
+   :width: 45%
+   :alt: Handle.Net configuration settings
+
+Fields:
+
+* **Web service endpoint**: The URL for (POST) requests to create and resolve
+  PIDs.
+* **Web service key**: Web service key needed for authentication to make
+  PID-binding requests to the PID web service endpoint.
+* **Naming authority**: Handle naming authority (e.g., 12345)
+* **Resolver URL**: The URL to append generated PIDs to in order to create
+  (potentially qualified) PURLs (persistent URLs) that resolve to the applicable
+  resolve URL. Note the second "r" in "resolver"!
+* **AIP PID source**: The source of the AIP's persistent identifier. The UUID of
+  the AIP is the default since it is virtually guaranteed to be unique. However,
+  the accession number of the transfer may be used, assuming the user can
+  guarantee a 1-to-1 relationship between the transfer and the AIP.
+* **Verify SSL certificates**: Selecting this box will ensure that Archivematica
+  verifies SSL certificates when making requests to bind PIDs.
+* **Archive resolve URL template**: Template (Django or Jinja2) for the URL that
+  a unit's PURL should resolve to. Has access to "pid" and "naming_authority"
+  variables.
+* **METS resolve URL template**: Template (Django or Jinja2) for the URL that a
+  unit's PURL with the "mets" qualifier should resolve to. Has access to "pid"
+  and "naming_authority" variables.
+* **File resolve URL template**: Template (Django or Jinja2) for the URL
+  that a file's PURL should resolve to. Has access to "pid" and
+  "naming_authority" variables.
+* **Access derivative resolve URL template**: Template (Django or Jinja2) for
+  the URL that a file's PURL with the "access" qualifier should resolve to. Has
+  access to "pid" and "naming_authority" variables.
+* **Preservation derivative resolve URL template**: Template (Django or Jinja2)
+  for the URL that a file's PURL with the "preservation" qualifier should
+  resolve to. Has access to "pid" and "naming_authority" variables.
+* **Original file resolve URL template**: Template (Django or Jinja2) for the
+  URL that a file's PURL with the "original" qualifier should resolve to. Has
+  access to "pid" and "naming_authority" variables.
+* **PID/handle request request body template**: Template (Django or Jinja2) that
+  constructs the HTTP request body using the rendered URL templates above. Has
+  access to the following variables: "pid", "naming_authority",
+  "base_resolve_url", and "qualified_resolve_urls", the last of which is a list
+  of dicts with "url" and "qualifier" keys.
```

Comment:
I think maybe we should provide an example of this form with all of the fields filled out with working values. With private and client-identifiable details redacted of course. This is pretty complicated to setup so I think an example would help. I will provide one ...

Reply:
@sallain this set of examples for the template fields may or may not be helpful.

Archive resolve URL template **example**:

    https://access.myinstitution.org/dip/{{ naming_authority }}/{{ pid }}

METS resolve URL template **example**:

    https://access.myinstitution.org/mets/{{ naming_authority }}/{{ pid }}

File resolve URL template **example**:

    https://access.myinstitution.org/access/{{ naming_authority }}/{{ pid }}

Access derivative resolve URL template **example**:

    https://access.myinstitution.org/access/{{ naming_authority }}/{{ pid }}

Preservation derivative resolve URL template **example**:

    https://access.myinstitution.org/preservation/{{ naming_authority }}/{{ pid }}

Original file resolve URL template **example**:

    https://access.myinstitution.org/original/{{ naming_authority }}/{{ pid }}

PID/handle request request body template **example**:

    <?xml version='1.0' encoding='UTF-8'?>
    <soapenv:Envelope
      xmlns:soapenv='http://schemas.xmlsoap.org/soap/envelope/'
      xmlns:pid='http://pid.myinstitution.org/'>
      <soapenv:Body>
        <pid:UpsertPidRequest>
          <pid:na>{{ naming_authority }}</pid:na>
          <pid:handle>
            <pid:pid>{{ naming_authority }}/{{ pid }}</pid:pid>
            <pid:locAtt>
              <pid:location weight='1' href='{{ base_resolve_url }}'/>
              {% for qrurl in qualified_resolve_urls %}
                <pid:location
                  weight='0'
                  href='{{ qrurl.url }}'
                  view='{{ qrurl.qualifier }}'/>
              {% endfor %}
            </pid:locAtt>
          </pid:handle>
        </pid:UpsertPidRequest>
      </soapenv:Body>
    </soapenv:Envelope>